modules/cli-administrator-security-policy.adoc

```diff
@@ -60,3 +60,8 @@ $ oc adm policy add-cluster-role-to-user edit user1
 ----
 $ oc adm policy add-scc-to-user privileged -z myserviceaccount
 ----
+
+[NOTE]
+====
+As of OpenShift 4.5, role-based access control (RBAC) objects are used manage security context constraints (SCC). Administrators create rolebindings for known cluster roles to update the SCC to a service account. See the documentation on https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html#role-based-access-to-ssc_configuring-internal-oauth[role-based access to security context constraints] for more information.
```

Comment:
```suggestion
As of {product-title} 4.5, role-based access control (RBAC) objects are used to manage security context constraints (SCC). Administrators create role bindings for known cluster roles to update the SCC to a service account. For more information, see xref:../../authentication/managing-security-context-constraints.adoc#role-based-access-to-ssc_configuring-internal-oauth[role-based access to security context constraints].
```
More info on these suggestions:
* Almost all instances of just "OpenShift" should be {product-title} (which resolves to "OpenShift Container Platform" for OCP docs
* We don't ever use hard links to docs.openshift.com (because we publish to the portal as well). It should always be an xref.
* xrefs should always back up (../) as far as necessary to the top level of the repo and then go back down. Also should always use the ".adoc" extension

Also a reminder not to use the "Commit suggestion" button that GitHub offers for this :)